src/core/connection.c

```diff
@@ -1635,7 +1635,7 @@ QuicConnTryClose(
         } else {
             QuicTraceEvent(
                 ConnTransportShutdown,
-                "[conn][%p] Transport Shutdown: %llu (Remote=%hhu) (QS=%hhu)",
+                "[conn][%p] Transport Shutdown: %u (Remote=%hhu) (QS=%hhu)",
```

Comment:
I think you're going to have to undo this. The is a breaking change to the logs.

Reply:
Forgot to revert this one. Got it now